tool/tbot/main.go

```diff
@@ -57,6 +57,8 @@ func Run(args []string, stdout io.Writer) error {
 	app.Flag("config", "Path to a configuration file.").Short('c').StringVar(&cf.ConfigPath)
 	app.HelpFlag.Short('h')
 
+	joinMethodList := fmt.Sprintf("%s", config.SupportedJoinMethods)
```

Comment:
Should we use `strings.Join` instead of `%s` to get a more user-friendly message?

Reply:
Good idea.

Reply:
773926ea1